api/src/pcapi/core/search/backends/algolia.py

```diff
@@ -418,6 +419,10 @@ def serialize_offer(cls, offer: offers_models.Offer, last_30_days_bookings: int)
             except AttributeError:
                 macro_section = None
 
+        product_extra_data = offer.product.extraData
+        gtl_code = product_extra_data.get("gtl_id") if product_extra_data else None
```

Comment:
(détail / style) Un peu plus facile à lire (et c'est ce qu'on fait plus haut pour `offer.extraData` : 

```suggestion
        product_extra_data = offer.product.extraData or {}
        gtl_code = product_extra_data.get("gtl_id")
```